third_party/boringssl/src/ssl/ssl_session.cc

```diff
@@ -1144,6 +1139,11 @@ void SSL_SESSION_get0_peer_sha256(const SSL_SESSION *session,
   }
 }
 
+int SSL_SESSION_early_data_capable(const SSL_SESSION *session) {
```

Comment:
Why move this in this PR?

Reply:
I moved it to a different namespace—before it was in a namespace that didn't correspond to the header, but it didn't matter that the function wasn't defined because it wasn't used in anything compiled. With us now compiling more unit tests, one/some of them use this function and it now would complain about a missing definition if I didn't move it